doc/release_notes/v2.1.0.rst

```diff
@@ -39,13 +39,15 @@ Enhancements
     *Waveform Sequence* element.
 * When JPEG 2000 image data is unsigned and the *Pixel Representation* is 1
   the image data is converted to signed (:issue:`1149`)
+* Added :attr:`~pydicom.uid.UID.keyword` property for the new UID keywords
+  in version 2020d of the DICOM Standard
 
 Changes
 .......
 * Reading and adding unknown non-private tags now does not raise an exception
   per default, only when :attr:`~pydicom.config.enforce_valid_values` is set
   (:issue:`1161`)
-* Data dictionaries updated to version 2020c of the DICOM Standard
+* Data dictionaries updated to version 2020d of the DICOM Standard
 
```

Comment:
Perhaps should also warn here that some UID names have changed, e.g.:

    '...Multiframe...' -> '...MultiFrame...'
    '...and...' -> '...And...'

Similarly the one DICOM dict 'ThreatDetectionAlgorithmAndVersion'

Reply:
Agreed, there are quite a few changes... I remember the hassle last time they did many changes (like `PatientsName` -> `PatientName`); I had to write a compatibilty layer for dcmtk (with deprecation warnings) to still allow the old names to avoid breaking a lot of client code. I don't think we need that, but there definitely shall be a respective note / warning in the release notes.